src/Gaffer/Metadata.cpp

```diff
@@ -497,8 +630,9 @@ void Metadata::registerValue( IECore::TypeId ancestorTypeId, const StringAlgo::M
 
 void Metadata::registerValue( IECore::TypeId ancestorTypeId, const StringAlgo::MatchPattern &plugPath, IECore::InternedString key, PlugValueFunction value )
 {
-	auto &graphComponentMetadata = graphComponentMetadataMap()[ancestorTypeId];
-	auto &plugValues = graphComponentMetadata.plugPathsToValues[StringAlgo::matchPatternPath( plugPath, '.' )];
+	auto &m = graphComponentMetadataMap()[ancestorTypeId];
+	const StringAlgo::MatchPatternPath matchPatternPath = StringAlgo::matchPatternPath( plugPath, '.' );
+	auto &plugValues = m.plugPathsToValues[matchPatternPath];
```

Comment:
I find it a bit confusing where we're referring to MatchPatterns as "plug path".  plugPathsToValues maps from MatchPatterns that could match an arbitrary number of plugs to values, correct?  This naming isn't new, but I figured I'd mention I found it confusing.

Reply:
Yeah, that makes sense. I suspect there's some history here where we didn't support match patterns at all to start with so "plug path" made more sense. Rather than introduce more churn at this point, I'll bear it in mind for any future refactor. I had a bit of a tricky time reacquainting myself with the code this time round, and I'm sure clarity could be improved in several places.